pulp_container/app/tasks/sync_stages.py

```diff
@@ -459,18 +459,35 @@ async def run(self):
         """
         Relate each item in the input queue to objects specified on the DeclarativeContent.
         """
-        async for dc in self.items():
-
-            if dc.extra_data.get('relation'):
-                self.relate_manifest_to_list(dc)
-            elif dc.extra_data.get('blob_relation'):
-                self.relate_blob(dc)
-            elif dc.extra_data.get('config_relation'):
-                self.relate_config_blob(dc)
-            elif dc.extra_data.get('man_relation'):
-                self.relate_manifest_tag(dc)
-
-            await self.put(dc)
+        async for batch in self.batches():
+            manifest_to_list_list = []
+            blob_list = []
+            config_blob_list = []
+            for dc in batch:
+                if dc.extra_data.get('relation'):
+                    manifest_to_list = self.relate_manifest_to_list(dc)
+                    manifest_to_list_list.append(manifest_to_list)
+                elif dc.extra_data.get('blob_relation'):
+                    blob = self.relate_blob(dc)
+                    blob_list.append(blob)
+                elif dc.extra_data.get('config_relation'):
+                    config_blob = self.relate_config_blob(dc)
+                    config_blob_list.append(config_blob)
+                elif dc.extra_data.get('man_relation'):
+                    self.relate_manifest_tag(dc)
+
+            ManifestListManifest.objects.bulk_create(objs=manifest_to_list_list,
+                                                     ignore_conflicts=True,
```

Comment:
Why is black not removing the hanging indent here?

Reply:
I see now, you made the PR against master. Should it be 2.0? @dkliban

Reply:
I changed the indentation and replaced all single quotes with double quotes to prevent possible conflicts when this commit will be merged into 2.0 as well.

Reply:
That is in all cases a good idea. Thank you.
I still want to settle the question whether this goes into 2.0 or master.

Reply:
We should merge into master and cherry-pick into 2.0 for now.